### PR TITLE
FHIR Uploader

### PR DESCRIPTION
**Pull request by ivan-c**
- Add `upload.py` util ([adapted from Paul's work in carl](https://github.com/uwcirg/carl/blob/main/carl/serialized/upload.py))
  - `./upload.py $FHIR_SERVER_URL` to upload all FHIR `.json` resources in `src/fhir`
  - `./upload.py $FHIR_SERVER_URL $FILE` to upload a single file

### CODE REVIEW

File: src/fhir/upload.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    tx_bundle = {
+        "resourceType": "Bundle",
+        # "id": "TBD",
+        "type": "transaction",
+        "entry": [],
+    }
+
+    for fhir_resource in fhir_resources:
+        if "id" not in fhir_resource:
+            print("skipping: ", fhir_resource)
```
Comment (pbugni) [inline src/fhir/upload.py:47]: perhaps this is just debugging, but it might be nice to know why it was skipped (i.e. no `id` attribute)

**Review by pbugni — APPROVED** [review]
Slick - I like it.
Any HAPI/FHIR config required to be able to PUT a resource by ID prior to its existence?

**Comment by mcjustin** [comment]
Since the CarePlan POST failed due to "Resource Patient/TBD not found...", perhaps we want to seed this w/ a Patient too? Ack that we're aiming to replace it w/ a patient-agnostic PlanDefinition once we have a route to $apply, but that's probably a ways out...

**Comment by pbugni** [comment]
Not certain where we landed on this - still feels like a general need of clients and or fEMR to be able to submit any number of FHIR files (in order using the alphabetical filename trick is reasonable).  Probably best to run with this per client implementation for now, and refactor into a separate service/library when the need/use is more obvious?

**Pull request merged**